website/content/docs/platform/k8s/vso/examples.mdx

```diff
@@ -11,6 +11,19 @@ The Operator project provides the following examples:
 - Sample use-cases are documented [here](https://github.com/hashicorp/vault-secrets-operator#samples)
 - A Terraform based demo can be found [here](https://github.com/hashicorp/vault-secrets-operator/tree/main/demo)
 
+## JWT auth for Kubernetes clusters in private networks
+
+Vault Secrets Operator supports using the [JWT auth method](/vault/docs/platform/k8s/vso/api-reference#vaultauthconfigjwt).
+In the default JWT configuration this involves an API call from the Vault server back to the Kubernetes API server to [verify](/vault/docs/auth/jwt#jwt-verification) the JWT.
+It is possible to configure Vault to use [`jwt_validation_pubkeys`](/vault/api-docs/auth/jwt#jwt_validation_pubkeys) fields of the JWT auth configuration
+to work in the Kubernetes environment and avoid the API call back to Kubernetes, removing the necessity to expose the Kubernetes API server externally.
+To configure this please follow the steps outlined for [Using JWT validation public keys](/vault/docs/auth/jwt/oidc-providers/kubernetes#using-jwt-validation-public-keys)
```

Comment:
Just want to double-check the ask on this, I don't believe the vaultauth CR would change based on enabling this configurable since that is purely on the vault side when configuring `auth/jwt/config`?
Is the ask to include some instruction on how to craft the JWT that is being referenced in the vaultauth CR? That is *sorta* covered in the link above [here](https://developer.hashicorp.com/vault/docs/auth/jwt/oidc-providers/kubernetes#creating-a-role-and-logging-in), where they use the service account token as the JWT.
Thoughts?

Reply:
Oh true, good point. I was just thinking given the title of the page it would be nice to have a VaultAuth CR example quite prominent that people can copy and edit. But you're right it's not really specific to this section. Happy to defer that if you prefer.

Reply:
Yeah I agree on your point about this being in an examples section so we should have an example.
This basically boils down to the fact that we don't have a JWT auth page yet and really this probably belongs there instead. I'm happy to take on moving this to the JWT page once it gets built, or work on the JWT page at some point.